include/model.php

```diff
@@ -797,7 +797,11 @@ public function filter_language_terms_orderby( $orderby, $args, $taxonomies ) {
 			return $orderby;
 		}
 
-		return sprintf( '%1$s.term_group, %1$s.term_id', $matches['alias'] );
+		/**
+		 * Makes the all terms in 'language' taxonomy first in the returned result by respecting the 'term_group' order.
+		 * The goal is not to mix terms between all language taxomonomies.
+		 */
```

Comment:
`/**` should be used for doc blocks only. I suggest to merg this comment with th method description.